blog/20250108-introducing-geojupyter/index.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Introducing GeoJupyter"
+date: "2025-01-08"
+description: |
+  We believe that geospatial data analysis and exploration should be free, open source,
+  and accessible for everyone!
+author:
+  - name: "Matt Fisher"
+    orcid: "0000-0003-3260-5445"
+categories:
+  - "Announcements"
+---
+
+## :jigsaw: Need
+
+**We believe that geospatial data analysis and exploration should be free, open source,
+and accessible for everyone!**
+
+Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
+ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
+programming-based GIS world which provides cutting-edge editing, reproducibility, and
+automated publishing experiences.
+
+Between those two worlds, users of programming-based workflows face the daunting problem
+of a blank page and steep learning curves, while non-coders are missing out on the
+power of programming-based workflows to enable flexible alterations, version control,
+and low-friction reproducibility and publishing.
+
+
+## :telescope: Vision
+
+![An animation from the 1990s childrens' TV program "Captain Planet" depicting the planeteers combining their powers to summon... Captain Planet!](https://media1.tenor.com/m/hWtovfyLjNAAAAAd/captain-planet-let-our-powers-combine.gif)
+
+<div class="quote">
+{{< include /elevator-pitch.md >}}
+</div>
+
+GeoJupyter will comprise a new generation of tools and enhancements to existing tools
+supporting exploring data about our planet. This includes tasks traditionally done
```

Comment:
Sentence read a little funny, maybe something like this?

```suggestion
GeoJupyter comprises of a new generation of tools and enhancements to existing
libraries supporting data exploration of our planet. This includes tasks traditionally done
```

Reply:
Thanks!

I think "comprises of" is not correct; would you prefer "consists of" or "comprises"?

> supporting data exploration of our planet

Feels like not quite there yet; what about  "supporting exploration of data about our planet"?

Reply:
> I think "comprises of" is not correct; would you prefer "consists of" or "comprises"?

Let's go with 'consists of'. I think you're correct that it should be just 'comprises' if I'm reading https://dictionary.cambridge.org/grammar/british-grammar/consist-comprise-or-compose correctly, but it reads strangely (at least to me) if you say 'GeoJupyter comprises a new generation of ...'.

> Feels like not quite there yet; what about "supporting exploration of data about our planet"?

Yep, that works too.

Reply:
>it reads strangely (at least to me) if you say 'GeoJupyter comprises a new generation of ...'

This is super useful feedback, thanks! That's a good enough reason to change it :)

Reply:
0d118ce
22d84ba